src/components/myReview/ReviewCard.tsx

```
@@ -24,41 +24,51 @@
-      <div className="imgContainer">
-        <img src={review.imgSrc} alt="review" />
+      <div
+        css={css`
+          display: flex;
+          gap: 10px;
+        `}
+      >
+        {Array.isArray(review.imgSrc) &&
+          review.imgSrc.length > 0 &&
+          review.imgSrc.map((src, index) => (
+            <div className="imgContainer">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)